src/app/legacy/psammead/psammead-headings/src/index.test.jsx

```diff
@@ -43,11 +46,14 @@ describe('SubHeading component', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it('should render correctly in dark mode', () => {
+  it('should render correctly on page types that support a dark ui', () => {
     const { container } = render(
-      <SubHeading script={latin} service="news" darkMode>
+      <SubHeading script={latin} service="news">
         This is a SubHeading
       </SubHeading>,
+      {
+        pageType: 'mediaArticle',
```

Comment:
Just thought, would it be better to import the page types from the [constants util file](https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/pageTypes.js) in tests?

Reply:
Good shout, I'll update those refs.